packages/components/src/components/textarea/textarea.stories.ts

```diff
@@ -30,237 +32,291 @@ export default {
+      ></sd-textarea>
+    </div>
+  `
+};
+
+/**
+ * Use the `rows` attribute to set the number of rows.
+ */
+
+export const Rows = {
+  name: 'Rows',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        placeholder="Placeholder example"
+        rows="4"
+        spellcheck=""
+      ></sd-textarea>
+
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        placeholder="Placeholder example"
+        rows="8"
+        spellcheck=""
+      ></sd-textarea>
+
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        placeholder="Placeholder example"
+        rows="12"
+        spellcheck=""
+      ></sd-textarea>
+    </div>
+  `
 };
 
 /**
- * Use the disabled attribute to disable a textarea. All interaction is disabled and no events will be fired.
+ * Use the `disabled` attribute to disable the textarea.
  */
 export const Disabled = {
-  parameters: {
-    controls: {
-      exclude: ['disabled']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'disabled', value: true }],
-          args
-        })}
-      </div>
-    `;
-  }
+  name: 'Disabled',
+  render: () => html`
+    <div class="w-[500px]">
+      <sd-textarea
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        placeholder="Placeholder"
+        rows="4"
+        spellcheck=""
+        disabled
+      ></sd-textarea>
+    </div>
+  `
 };
 
 /**
- * Use the readonly attribute to render a textarea as readonly.  Interaction is enabled, but the textarea cannot be edited.  Events will be fired.
+ * Use the `readonly` attribute to render a textarea as readonly.
+ *
+ * Interaction is enabled, but the textarea cannot be edited.
+ * Events will be fired.
  */
 export const Readonly = {
-  parameters: {
-    controls: {
-      exclude: ['readonly']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'readonly', value: true }],
-          args
-        })}
-      </div>
-    `;
-  }
+  name: 'Readonly',
+  render: () => html`
+    <div class="w-[500px]">
+      <sd-textarea
+        value="value"
+        size="lg"
+        label="Label"
+        help-text="help-text"
+        rows="4"
+        readonly=""
```

Comment:
Is this set to true?
Because if read-only is true then the field background should be neutral-100
[See figma](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2619-19572&t=zNWQRZ1BICG0FZXo-11)

Reply:
this was a small css bug, I made the fix and pushed it here. IMO this is okay. But want to confirm with @mariohamann.